docs/examples/compute/digitalocean/instantiate_api_v1.0.py

```diff
@@ -2,4 +2,4 @@
 from libcloud.compute.providers import get_driver
 
 cls = get_driver(Provider.DIGITAL_OCEAN)
-driver = cls('client id', 'api key', api_version='v1')
```

Comment:
Hey, sorry for the delay. Personal life had some crazy changes cutting my time for projects to almost nothing. Going over these, and they should be easy to get done over the next couple days.

As for this entry in particular, I agree it should be explicit here as well. I'm also going to update the v2 API compute example to include the `api_version='v2'` as well.